scripts/qemu-wrapper

```diff
@@ -193,6 +194,17 @@ def main(argv):
             depriv = False
             continue
 
+        if p == "-serial":
+            params = qemu_args[n + 1].split(',')
+            # There are other network options apart from 'tcp' but,
+            # in practice, tcp is always used for serial debug
+            if params[0].startswith('tcp'):
+                print "serial: '%s'" % qemu_args[n + 1]
```

Comment:
Did you intend to keep this `print` here? I assume since the option itself is for debugging we can keep it if it makes debugging easier.